web3.js/src/connection.ts

```diff
@@ -755,6 +755,51 @@ export type BlockSignatures = {
   blockTime: number | null;
 };
 
+/**
+ * recent block production information
+ */
+export type BlockProduction = {
+  /** a dictionary of validator identities, as base-58 encoded strings. Value is a two element array containing the number of leader slots and the number of blocks produced */
+  byIdentity: Record<string, Array<number>>;
+  /** Block production slot range */
+  range: {
+    /** first slot of the block production information (inclusive) */
+    firstSlot: number;
+    /** last slot of block production information (inclusive) */
+    lastSlot: number;
+  };
+};
+
+/**
+ * Configuration object for getBlockProduction requests
+ */
+export type GetBlockProductionConfig = {
+  /** Optional commitment level */
```

Comment:
I agree that the comment is a little redundant. I added the comment because I saw similar comments in the codebase (Eg, `GetProgramAccountsConfig` on line 1803) and assumed that was the expected practice. Would you like me to remove the other comments in `connection.ts` like this one, too? (there are quite a few)

Reply:
I wouldn't mind, but I wouldn't want it to block this PR.

Reply:
Good point! I made a separate PR for the other comments here #23950

Marking this as resolved.